src/main/java/seedu/contax/logic/parser/RangeCommandParser.java

```diff
@@ -27,18 +29,26 @@ public RangeCommand parse(String args) throws ParseException {
 
         String commandInput = argMultimap.getPreamble();
 
-        Index fromIndex;
-        Index toIndex;
+        Index fromIndex = Index.fromZeroBased(0);
+        Index toIndex = Index.fromZeroBased(0);
```

Comment:
You should move this to after this guard clause.

Reply:
this is init only shd be same, but yes can move